diff --git a/src/DevHunter.Common/NotificationMessagesConstants.cs b/src/DevHunter.Common/NotificationMessagesConstants.cs
--- a/src/DevHunter.Common/NotificationMessagesConstants.cs
+++ b/src/DevHunter.Common/NotificationMessagesConstants.cs
@@ -8,0 +9,4 @@
+		public const string UnexpectedError = "Unexpected error occurred!";
+		public const string DevelopmentNotFoundError = "Development with the provided id does not exist!";
+		public const string TechnologyNotFoundError = "Technology with the provided id does not exist!";
+
diff --git a/src/DevHunter.Web/Controllers/CompanyController.cs b/src/DevHunter.Web/Controllers/CompanyController.cs
--- a/src/DevHunter.Web/Controllers/CompanyController.cs
+++ b/src/DevHunter.Web/Controllers/CompanyController.cs
@@ -56 +56 @@
-			TempData[ErrorMessage] = "Unexpected error occurred!";
+			TempData[ErrorMessage] = UnexpectedError;
diff --git a/src/DevHunter.Web/Controllers/HomeController.cs b/src/DevHunter.Web/Controllers/HomeController.cs
--- a/src/DevHunter.Web/Controllers/HomeController.cs
+++ b/src/DevHunter.Web/Controllers/HomeController.cs
@@ -67 +67 @@
-				TempData[ErrorMessage] = "Unexpected error occurred!";
+				TempData[ErrorMessage] = UnexpectedError;
diff --git a/src/DevHunter.Web/Controllers/JobOfferController.cs b/src/DevHunter.Web/Controllers/JobOfferController.cs
--- a/src/DevHunter.Web/Controllers/JobOfferController.cs
+++ b/src/DevHunter.Web/Controllers/JobOfferController.cs
@@ -188 +188 @@
-			TempData[ErrorMessage] = "Unexpected error occurred!";
+			TempData[ErrorMessage] = UnexpectedError;
diff --git a/src/DevHunter.Web/Areas/Admin/Controllers/DevelopmentController.cs b/src/DevHunter.Web/Areas/Admin/Controllers/DevelopmentController.cs
--- a/src/DevHunter.Web/Areas/Admin/Controllers/DevelopmentController.cs
+++ b/src/DevHunter.Web/Areas/Admin/Controllers/DevelopmentController.cs
@@ -95 +95 @@
-                    TempData[ErrorMessage] = "Development with the provided id does not exist!";
+                    TempData[ErrorMessage] = DevelopmentNotFoundError;
@@ -125 +125 @@
-			        TempData[ErrorMessage] = "Development with the provided id does not exist!";
+			        TempData[ErrorMessage] = DevelopmentNotFoundError;
@@ -152 +152 @@
-			        TempData[ErrorMessage] = "Development with the provided id does not exist!";
+			        TempData[ErrorMessage] = DevelopmentNotFoundError;
@@ -170 +170 @@
-            TempData[ErrorMessage] = "Unexpected error occurred!";
+            TempData[ErrorMessage] = UnexpectedError;
diff --git a/src/DevHunter.Web/Areas/Admin/Controllers/TechnologyController.cs b/src/DevHunter.Web/Areas/Admin/Controllers/TechnologyController.cs
--- a/src/DevHunter.Web/Areas/Admin/Controllers/TechnologyController.cs
+++ b/src/DevHunter.Web/Areas/Admin/Controllers/TechnologyController.cs
@@ -115 +115 @@
-					TempData[ErrorMessage] = "Technology with the provided id does not exist!";
+					TempData[ErrorMessage] = TechnologyNotFoundError;
@@ -145 +145 @@
-					TempData[ErrorMessage] = "Technology with the provided id does not exist!";
+					TempData[ErrorMessage] = TechnologyNotFoundError;
@@ -173 +173 @@
-					TempData[ErrorMessage] = "Technology with the provided id does not exist!";
+					TempData[ErrorMessage] = TechnologyNotFoundError;
@@ -207 +207 @@
-			TempData[ErrorMessage] = "Unexpected error occurred!";
+			TempData[ErrorMessage] = UnexpectedError;
diff --git a/src/DevHunter.Web/Areas/Company/Controllers/CompanyController.cs b/src/DevHunter.Web/Areas/Company/Controllers/CompanyController.cs
--- a/src/DevHunter.Web/Areas/Company/Controllers/CompanyController.cs
+++ b/src/DevHunter.Web/Areas/Company/Controllers/CompanyController.cs
@@ -147 +147 @@
-            TempData[ErrorMessage] = "Unexpected error occurred!";
+            TempData[ErrorMessage] = UnexpectedError;
diff --git a/src/DevHunter.Web/Areas/Company/Controllers/JobOfferController.cs b/src/DevHunter.Web/Areas/Company/Controllers/JobOfferController.cs
--- a/src/DevHunter.Web/Areas/Company/Controllers/JobOfferController.cs
+++ b/src/DevHunter.Web/Areas/Company/Controllers/JobOfferController.cs
@@ -176 +176 @@
-			TempData[ErrorMessage] = "Unexpected error occurred!";
+			TempData[ErrorMessage] = UnexpectedError;
diff --git a/src/DevHunter.Web/Areas/Manage/Controllers/AccountController.cs b/src/DevHunter.Web/Areas/Manage/Controllers/AccountController.cs
--- a/src/DevHunter.Web/Areas/Manage/Controllers/AccountController.cs
+++ b/src/DevHunter.Web/Areas/Manage/Controllers/AccountController.cs
@@ -54 +54 @@
-                TempData[ErrorMessage] = "Unexpected error occurred!";
+                TempData[ErrorMessage] = UnexpectedError;
@@ -88 +88 @@
-                TempData[ErrorMessage] = "Unexpected error occurred!";
+                TempData[ErrorMessage] = UnexpectedError;
